src/validation/rules/ExecutableDefinitions.js

```diff
@@ -27,23 +30,24 @@ export function ExecutableDefinitions(
   context: ASTValidationContext,
 ): ASTVisitor {
   return {
-    Document(node) {
-      for (const definition of node.definitions) {
-        if (!isExecutableDefinitionNode(definition)) {
```

Comment:
@mjmahone I was searching for non-SDL examples of iterating over all definition pattern and found this 😄